src/Pages/CalculatorPage.jsx

```diff
@@ -39,8 +39,8 @@ const CalculatorPage = () => {
                         buttonHref={buttons[0].href}
                         tooltip={tooltip}
                         progressBar={progressBar}
-                        // currentSectionName={progressBar.currentSectionName}
-                        // totalSections={progressBar.totalSections}
+                    // currentSectionName={progressBar.currentSectionName}
```

Comment:
It doesn't look like we are using these props anymore. I know you didn't write this code but since we're in this file, let's take this opportunity to remove the commented out code :)